examples/tts/conf/talknet-pitch.yaml

```diff
@@ -55,11 +60,12 @@ model:
         punct: true
         spaces: true
         stresses: false
+        add_blank_at: last
     dataloader_params:
       drop_last: false
       shuffle: false
       batch_size: 64
-      num_workers: 8
+      num_workers: 1
```

Comment:
having single worker doesn't really make sense, does it?
num_workers=0 should work at least as fast

Reply:
In my experiments, num_workers=1 works slightly faster

Reply:
could be random fluctuation
num_workers=0 does not spawn processes
num_workers=1 does spawn just single process, which does exactly the same thing